src/test/kotlin/racingcar/util/FakeRandomGenerator.kt

```diff
@@ -0,0 +1,7 @@
+package racingcar.util
+
+import racingcar.domain.RacingGame.Companion.MIN_VALUE_TO_MOVE_CAR
+
+class FakeRandomGenerator : RandomGenerator {
+    override fun generate(range: IntRange): Int = MIN_VALUE_TO_MOVE_CAR
+}
```

Comment:
generate되는 숫자를 파라미터로 받으면 원하는 값을 테스트할때 용이할거 같아요 :)
```suggestion
class FakeRandomGenerator(private val generateNumber: Int) : RandomGenerator {
    override fun generate(range: IntRange): Int = generateNumber
}
```